jib-core/src/test/java/com/google/cloud/tools/jib/event/events/ProgressEventTest.java

```diff
@@ -117,6 +118,22 @@ public void testSmoke() {
     Assert.assertEquals(2, allocationCompletionMap.get(AllocationTree.root).longValue());
   }
 
+  @Test
+  public void testType() {
```

Comment:
Note that this test would also need to check if the consumers were actually called.